syntax = "proto3";

package rpc_agent;

message CreateRingRequest{
	string name = 1;
}

message Errno{
	int32 err = 1;
}


service RPC {
	rpc rpc_create_ring(CreateRingRequest) returns (Errno);

}
